docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Work with workbooks using the Excel JavaScript API
+description: ''
+ms.date: 11/26/2018
+---
+
+
+# Work with Workbooks using the Excel JavaScript API
+
+This article provides code samples that show how to perform common tasks with workbooks using the Excel JavaScript API. For the complete list of properties and methods that the **Workbook** object supports, see [Workbook Object (JavaScript API for Excel)](https://docs.microsoft.com/javascript/api/excel/excel.workbook). This article also covers workbook-level actions performed through other similarly scoped objects, such as [Application](https://docs.microsoft.com/javascript/api/excel/excel.application).
```

Comment:
I find the phrase "similarly scoped" mysterious, and I don't think its needed. Can we just delete it?